Integrations/integration-CounterTack.yml

```diff
@@ -85,8 +95,10 @@ script:
             auth=(USERNAME, PASSWORD),
             headers=headers,
             verify=USE_SSL,
-            data=body
+            data=body,
+            params=params
```

Comment:
please remove `params` if never used

Reply:
params is used in fetch incidents command (get behaviors)